content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+- #2013: Fixed some ANSI escape codes in the Italian translation
+
+- #2118: Update Polish (pl) translation
+- #2098: Update Polish (pl) translation
+- #2070: Update Polish (pl) translation
+- #2012: Update Polish (pl) translation, change ANSI escape codes to tags
+- #1982: Update Polish (pl) translation
+
+- #2120: Update Russian (ru-RU) translation
+
+- #2064: Cleanup string IDs of DOS programs
+- #2126: Improve build instructions
+- #2094: Fix character D6 mapping in code page 3021 (Bulgarian MIK encoding)
+- #2074: Clarify `tandy` and `ibm5153` `cga_colors` config descriptions
+- #2036: Add release drafter support files
+- #2032: Translate new message
+- #2017: Fix mistakes in switching to ansi text codes
+- #2011: Adapt 'update-sources.sh' script to UTF-8 locale
+- #2008: convert esc codes to ansi in src
+- #1992: Improve detection of atomics, testing of speexDSP, and cleanup README 
+- #2001: Parse language options before writing the conf file
+- #1988: Fix code page 30020 to UTF-8 mapping
+- #1984: Switch messages in MSG_Add from code page 437 to UTF-8
+- #1884: Support for UTF-8 locale (2nd generation patch)  
+- #2082: Add handling for ABNT2 slash/question mark key (by rsribeiro)
+- #2058: Delete obsolete translated msgs
+- #1984: Switch messages in MSG_Add from code page 437 to UTF-8
+- #1884: Support for UTF-8 locale (2nd generation patch)
+- #1988: Fix code page 30020 to UTF-8 mapping
+</details>
+
+##  8. <a name='Documentationupdates'></a>Documentation updates
+
+The following wiki articles have been updated:
+
+- The new [Dual Mouse Gaming
+  page](https://github.com/dosbox-staging/dosbox-staging/wiki/Dual-Mouse-Gaming)
+  documents how to setup dual-mouse working in Settlers 1 and II.
+
+- The [Windows 3.1x
+  page](https://github.com/dosbox-staging/dosbox-staging/wiki/Windows-3.1x) was
+  updated with notes on how to setup the improved mouse driver, and more sound
+  drivers.
+
+- The new [Windows-3.0
+  page](https://github.com/dosbox-staging/dosbox-staging/wiki/Windows-3.0)
+  documents how to setup Windows 3.0's video, sound, MIDI, mouse, and how to
+  print-to-file. 
+
+ - The new [Serial Ports
+   page](https://github.com/dosbox-staging/dosbox-staging/wiki/Serial-Ports)
+   documents all of the serial port settings.
+
+ - The [Sound Cards
+   page](https://github.com/dosbox-staging/dosbox-staging/wiki/Sound-Cards) was
+   overhauled with descriptions and configuration help for all supported sound
+   devices.
+
+- The [Tips and Tricks
+  page](https://github.com/dosbox-staging/dosbox-staging/wiki/Tips-&-Tricks) was
+  updated with how to generating DOSBox mountable CD-DA content from **public
+  domain** or **[Creative Commons]**-licensed YouTube chapters and playlists.
+
+##  9. <a name='Contributors'></a>Contributors
+
+The following commit authors, sponsors, supports, and backers all invested their
+time or funds into the 0.80 release effort. Thank you!
+
+ * **0.80 Commit authors (alphabetic order):**
```

Comment:
Use the next heading, `###` in this case, and remove the `:` from the end.

Same comment for the remaining headings.

Reply:
Looks good; thanks!

Reply:
Lower case the title: 0.80 commit authors